### PR TITLE
Modify readme and fix save privaty key on history bug

### PR DESCRIPTION
**Pull request by driftluo**
1. Modify readme
2. Fix save privaty key on history bug

### CODE REVIEW

File: cita-cli/src/interactive.rs [hunk 1]
```diff
@@ -424,3 +424,19 @@ impl GlobalConfig {
...
+            .split_whitespace()
+            .filter(|key| !(key.starts_with("0x") && key.len() == 128 + 2))
+            .collect::<Vec<&str>>()
+            .join(" ")
+    } else {
+        private_key
+            .split_whitespace()
+            .filter(|key| !(key.starts_with("0x") && key.len() == 64 + 2))
+            .collect::<Vec<&str>>()
+            .join(" ")
```
Comment (TheWaWaR) [inline cita-cli/src/interactive.rs]: The difference is just `64` and `128` why write the main logic twice?

Comment (driftluo) [reply cita-cli/src/interactive.rs]: Just copy .... emmm, I will change it immediately

File: cita-cli/src/interactive.rs [hunk 2]
```diff
@@ -424,3 +424,12 @@ impl GlobalConfig {
         println!("{}", output);
     }
 }
+
+fn remove_private(private_key: &str, blake_2b: bool) -> String {
+    let len = if blake_2b { 128 + 2 } else { 64 + 2 };
+    private_key
+        .split_whitespace()
```
Comment (TheWaWaR) [inline cita-cli/src/interactive.rs]: Please use `shell_words.split()`

What if the command is:
`rpc Xxx  --private-key "0xabcdef1230....."`

File: cita-cli/src/interactive.rs [hunk 3]
```diff
@@ -424,3 +424,12 @@ impl GlobalConfig {
         println!("{}", output);
     }
 }
+
+fn remove_private(private_key: &str, blake_2b: bool) -> String {
```
Comment (TheWaWaR) [inline cita-cli/src/interactive.rs]: The argument name:
`private_key` => `line`

**Pull request merged**